2001-3000/LIT2892RepCh173.xml

```diff
@@ -58,13 +59,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <bibl>
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">173</citedRange>
+                  <note>Chaine mistakenly attributed this malkǝʾ-hymn to Walatta Ṗeṭros in his Repertoire des Salam, but correctly attributed the hymn to ʾƎḫǝta Krǝstos in <bibl>
+                     <ptr target="bm:Chaine1912Catalogue"/>
+                     <citedRange unit="page">57</citedRange>
+                     <citedRange unit="number">88</citedRange>
+                  </bibl>
+                     .</note>
                </bibl>
             </listBibl>
          </div>
          <div type="edition">
             <div type="textpart" subtype="incipit" xml:lang="gez">
```

Comment:
Add a note with a hint to the manuscript or edition, from which this incipit was taken from, i.e.:

`<note>This incipit was taken from <ref type="mss" corresp="BNFabb88"/> according to <bibl><ptr target="bm:Chaine1912Catalogue"/><citedRange unit="page">57</citedRange><citedRange unit="number">88</citedRange></bibl></note>`